src/components/ProductDetail/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+function ProductDetailComponent() {
```

Comment:
How about renaming it `ProductView`? And inside this component should contain `ProductContent`, `ProductDetail`, and `ProductInformation`? Each styling class for each component. Keep the `grid` on `ProductView` and create new 3 components.

```
const ProductView = () => {
  return (
    <section className="product-view">
      <div className="product-view__content">
        <ProductContent />
      </div>
      <div className="product-view__detail">
        <ProductDetail />
      </div>
      <div className="product-view__information">
        <ProductInformation />
      </div>
    </section>
  )
}
```

Reply:
After discussing, `ProductView` is a better name. But the component should be `ProductImage`, `ProductInformation`, and `ProductDescription`. Content can be anything `image` is more specific from my view.

Reply:
## TL;DR

Prefix `image` is for an element while `content` is for a block. A component is a block which consist of elements.

## Explanation

Yep `ProductImage` is a good name. But seems like you forgot to think about the other essentials(i.e. `slider`, `dots`, `button--arrow`). If we name the component `ProductImage`, we have to name those elements as `.product-image__slider` or `.product-image__owl-dots`. This isn't meaningful enough and violates **BEM** convention. On the other hand, `.product-content__image`/`.product-content__button--arrow`/`.product-content__slider`.

What do you think?

Reply:
As you mentioned, content as a `block` why don't we use `ProductContent` instead of `ProductView` and `ProductView` for viewing image. TBH, I'd like something more specific than just `content` because those 3 elements could be named 'content' as well as I have done before. Anyway, `ProductDetail` is used for the `page` name already we should avoid this. How about this?
`const ProductContent = () => {
  return (
    <section className="product-content">
      <div className="product-content__view">
        <ProductView />
      </div>
      <div className="product-content__information">
        <ProductInformation />
      </div>
      <div className="product-content__description">
        <ProductDescription />
      </div>
    </section>
  )
}`

Reply:
Anyone would like to bring some ideas? @kimvcp @Kittinske15?

Reply:
Firstly, to discrimate details and information: https://www.quora.com/What-is-the-difference-between-details-and-information
Let's keep the word **description** for `<p>` and use **information** insteas.

Next, naming a component `ProductDetail` is not going to break the `prodct-detail.tsx` since the component were importing is `ProductView`.

Lastly, I dont think `.product-view__image`, `.product-view__owl-dots`, `.product-view__slider`, etc does make sense. The word **content** can be specifically use for image though.

Inspiration: https://stockx.com

(please kindly have a look)

Reply:
Agree with changing ``ProductDetail`` to ``ProductView`` in pages and also using ``product-content`` is more meaningful than ``product-view`` for my opinion.

Reply:
I personally like how `stockx` name their components. I'm pretty okay if we decide to follow them.

Reply:
Okay then, I'll keep with this version --> [https://github.com/Cobda/cobda-web/pull/6#discussion_r570333725]